core/workflow/aggregatorrole.go

```diff
@@ -76,15 +76,16 @@ func (r *aggregatorRole) GlobFilter(g glob.Glob) (rs []Role) {
 	return
 }
 
-func (r *aggregatorRole) ProcessTemplates() (err error) {
+
+func (r *aggregatorRole) ProcessTemplates(repo *repos.Repo) (err error) {
```

Comment:
For clarity I'd always call this argument to the various implementations of `ProcessTemplates` `workflowRepo` instead of just `repo`.

Reply:
Done